README.md

```diff
@@ -92,12 +117,120 @@ Tested on Ubuntu 22.04.2 LTS aarch64, 5.15.0-69-generic
 
 To apply the disk layout to a target machine, you'll need to boot the machine using the built image and obtain the `mounts.nix` file for that specific host. Once you have the file, execute the following command:
 
 ```
-sudo nix run github:nix-community/disko -- --mode create ./mounts.nix
+sudo nix run github:nix-community/disko -- --mode zap_create_mount ./mounts.nix
 ```
 
-This command will format the disks according to the script. Once formatting is complete, reboot the machine and the disks should be ready to use.
\ No newline at end of file
+This command will format the disks according to the script. Once formatting is complete, reboot the machine and the disks should be ready to use.
+
+## Formats & Deployment
+
+- kexecTree
+  
+  Outputs: bzImage, initrd, kexec-boot script, and netboot IPXE-script.
+  
+  Deploy:
+  
+  ```
+  # Run a bash shell that provides the dependencies
+  $ nix develop
+
+  # Run kexec-boot script
+  $ ./result/kexec-boot
+  ```
+
+  <details>
+  <summary>Netbooting Raspberry Pi 4 with UEFI Firmware</summary>
+
+    We'll be gathering the boot media (/tftpboot folder for PXE booting) in the `/result` directory. Make sure you have the following dependencies installed: docker, unzip.
+
+    Clone the project repository and build the EDK2 Raspberry Pi 4 UEFI firmware. 
+    ```
+    git clone https://github.com/valtzu/pipxe.git
+    cd pixpe
+    sudo docker-compose up
+    ```
+    
+    Create a result directory and copy the UEFI firmware files there.
+    ```
+    mkdir -p result
+    cp pxe/RPI_EFI.fd result
+    cp -r pxe/efi result
+    ```
+
+    Download the "standard" [RPi4 UEFI releases from Github](https://github.com/pftf/RPi4/releases) and copy the files (excluding RPI_EFI.fd) to the `result` directory.
+    ```
+    wget https://github.com/pftf/RPi4/releases/download/v1.34/RPi4_UEFI_Firmware_v1.34.zip
+    unzip RPi4_UEFI_Firmware_v1.34.zip -d result -x RPI_EFI.fd
+    ```
+
+    Obtain all firmware overlays from the [Raspberry Pi Github repository](https://github.com/raspberrypi/firmware/tree/master/boot/overlays) and add them to the overlays folder in the `result` directory. When prompted to override files, keep the `miniuart-bt.dtbo` and `upstream-pi4.dtbo` from the UEFI project.
+    ```
+    cp -n overlays/* result/tftpboot/overlays/
+    ```
+
+    Replace the autoexec.ipxe file in the projects folder with your own custom ipxe script.
+    ```
+    cat > result/efi/boot/autoexec.ipxe << EOF
+    #!ipxe
+    dhcp
+    chain --autofree http://192.168.1.128:8080/netboot.ipxe || shell
+    EOF
+    ```
+
+    Use rpi-imager to flash "Raspberry Pi OS Lite (32-bit)" to an SD card, boot from it, update the system, and change the boot order using `raspi-config` (Advanced Settings > Boot Order > Network Boot). Finally, remove the SD card and reboot.
```

Comment:
I wonder is the reason that USB boot (of an .ISO file) does not work is that the USB configuration is not mentioned here.

Reply:
These instructions are specifically for kexec-boot.

I changed the boot order to USB when attempting to boot the `isoImage` format, but it was just a quick test. I didn't put any effort into investigating it at the time. However, I'll give it another shot when I have the time. I doubt there are any major issues with this, or at least I hope so.

Now that I think about it, I'm not even sure if `dinar` hosts will boot from a physical USB drive since Tommi uses them through virtualization. More testing is needed in this regard.